clang/unittests/Format/FormatTest.cpp

```diff
@@ -29053,6 +29053,11 @@ TEST_F(FormatTest, KeywordedFunctionLikeMacros) {
                Style);
 }
 
+TEST_F(FormatTest, UnbalancedAngleBrackets) {
+  verifyNoCrash("typename foo<bar>::value, const String &>::type f();",
```

Comment:
For me you said, I should use `verifyFormat`, why not here?

Reply:
Because it's invalid code here, not just incomplete code.